pkg/build/api/register.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 const GroupName = ""
+const FutureGroupName = "build.openshift.io"
```

Comment:
Technically all of those nouns describe processes.  The word to describe a build process is "building".  Also "app" sounds wrong.  You generally don't pluralize processes, but you do pluralize groups of things.

Is there a rule that we are using singular nouns?  Or the rule is that we're describing what the group does / contains.